docs/sdk/server-side-sdks/node/node-usage.md

```diff
@@ -1,54 +1,13 @@
 ---
-title: Node.js SDK
-sidebar_position: 1
+title: DevCycle Node.js SDK Usage
+sidebar_label: Usage
+sidebar_position: 3
 ---
 
-# DevCycle NodeJS Server SDK
-
-The NodeJS Server SDK for DevCycle.
-
-This SDK uses local bucketing to perform all user segmentation and bucketing locally in the SDK, 
-providing immediate responses to variable and feature requests for a user. 
-
-The SDK will download the latest version of your DevCycle environments configuration from a CDN on initialization,
-and will periodically poll the CDN for configuration changes.
-
-The SDK is available as a package on npm. It is also open source and can be viewed on Github.
-
 [![Npm package version](https://badgen.net/npm/v/@devcycle/nodejs-server-sdk)](https://www.npmjs.com/package/@devcycle/nodejs-server-sdk)
 [![GitHub](https://img.shields.io/github/stars/devcyclehq/js-sdks.svg?style=social&label=Star&maxAge=2592000)](https://github.com/devcyclehq/js-sdks)
 
-## Installation
-
-Our library can be found on npm and installed by the following:
-
-```
-npm install --save @devcycle/nodejs-server-sdk
-```
-
-## Getting Started
-
-To use the DVC Server SDK in your project, import the `@devcycle/nodejs-server-sdk` package and 
-call `initialize` with your DVC SDK server key. You may optionally `await` for the client
-to be initialized.
-
-JS Example:
-```javascript
-const DVC = require('@devcycle/nodejs-server-sdk')
-
-const dvcClient = await DVC.initialize('<DVC_SDK_SERVER_KEY>').onClientInitialized()
-```
-
-Typescript Example:
-```typescript
-import { initialize } from '@devcycle/nodejs-server-sdk'
-
-const dvcClient = await initialize('<DVC_SDK_SERVER_KEY>').onClientInitialized()
-```
-
-## Usage
-
-### Initialization Options
+## Initialization Options
```

Comment:
should we put this with the Initialization steps in the Getting Started page?

Reply:
good idea, i'll move it